process/unsigned/interceptedUnsignedTransaction.go

```diff
@@ -111,20 +111,9 @@ func (inUTx *InterceptedUnsignedTransaction) processFields(uTxBuffWithSig []byte
 
 // integrity checks for not nil fields and negative value
 func (inUTx *InterceptedUnsignedTransaction) integrity() error {
```

Comment:
here too, the unexported function is no longer needed

Reply:
done